### DISCUSSION TITLE
What is the purpose of this library？

### DISCUSSION THREAD

**Discussion by zbindu** [phalcon/phalcon#412] [Q&A]
Hi, Can you explain what this library is used to be? I'm a little confused. Thanks.

**Comment by niden** [comment]
This is the next version of Phalcon (v6). We are converting the Zephir code from Phalcon v5 to PHP. 

It is not ready yet, since a few namespaces are missing but we plan on getting this ready this year.

**Comment by zbindu** [comment]
> This is the next version of Phalcon (v6). We are converting the Zephir code from Phalcon v5 to PHP.
> 
> It is not ready yet, since a few namespaces are missing but we plan on getting this ready this year.

Does this mean there will be a php version of phalcon in the future ? Will cphalcon still exist .

**Comment by niden** [comment]
Yes there will be a PHP version of Phalcon. A cphalcon will also exist where you will be able to load it on your web server and will accompany the PHP version, offering higher performance in areas of the framework that are slow

**Reply by Pigneguy89** [reply]
> Yes there will be a PHP version of Phalcon[.](https://youtubevanceds.org/) A cphalcon will also exist where you will be able to load it on your web server and will accompany the PHP version, offering higher performance in areas of the framework that are slow

That sounds promising! The PHP version of Phalcon combined with cPhalcon will definitely improve performance. Having the ability to load it directly on the web server for enhanced efficiency, especially in slower areas of the framework, is a great move. This will make Phalcon even more powerful and appealing for developers. Looking forward to it.
